modules/react-native-status/android/src/main/java/im/status/ethereum/module/AccountManager.kt

```diff
@@ -224,16 +224,25 @@ class AccountManager(private val reactContext: ReactApplicationContext) : ReactC
         val absRootDirPath = utils.getNoBackupDirectory()
         val newKeystoreDir = utils.pathCombine(absRootDirPath, "keystore")
 
+        val jsonParams = JSONObject()
+        jsonParams.put("keyStoreDir", newKeystoreDir)
+        jsonParams.put("address", address) 
+        jsonParams.put("password", password)
+
         utils.executeRunnableStatusGoMethod(
-            { Statusgo.verifyAccountPassword(newKeystoreDir, address, password) },
+            { Statusgo.verifyAccountPasswordV2(jsonParams.toString()) },
```

Comment:
Any reason for creating a json object then adding key value pairs to it as params and then converting it to a string while calling statusgo?

to me it seems simpler to just pass in the string parameters.

```suggestion
            { Statusgo.verifyAccountPasswordV2(newKeystoreDir, address, password) },
```

Reply:
also I couldn't find `verifyAccountPasswordV2 ` in status-go changes

Reply:
I did however find `VerifyDatabasePasswordV2`